active_learning/active_learning.py

```diff
@@ -1,21 +1,29 @@
-from util.datastructures import MetaPath
+from util.datastructures import MetaPath, MetaPathRatingGraph
 
 # mock data
 mock_paths = [MetaPath([1, 2], [3]), MetaPath([1, 2, 1], [3, 4]), MetaPath([1, 3, 2], [1, 1]),
               MetaPath([1, 3, 2], [1, 1]), MetaPath([1, 3, 2], [1, 1])]
 
 # algorithm types
-NO_RATING = 'no_rating'
+NO_USER_FEEDBACK = 'no_user_feedback'
```

Comment:
Think about adding this to the `util/config.py` file

Reply:
After reading further, I would put this in the config file. Maybe put a comment to it.

Reply:
I explicitly did not do this, cause this config is only internal to active learn. We will use the config verbs if it is plausible

Reply:
Ok, then I would suggest to switch to a real configuration parser like this one https://docs.python.org/3.6/library/configparser.html. It isn't very nice with this `config.py` anyways.